src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -730,16 +730,17 @@ subroutine ocn_diagnostic_solve(dt, statePool, forcingPool, meshPool, diagnostic
       !$omp do schedule(runtime) private(invAreaCell1, k, shearSquared, i, iEdge, factor, delU2, shearMean)
       do iCell=1,nCells
          RiTopOfCell(:,iCell) = 100.0_RKIND
-         invAreaCell1 = 1.0_RKIND / areaCell(iCell)
+         areaSum = 0.0_RKIND 
          do k=2,maxLevelCell(iCell)
            shearSquared = 0.0_RKIND
            do i = 1, nEdgesOnCell(iCell)
              iEdge = edgesOnCell(i, iCell)
-             factor = 0.5_RKIND * dcEdge(iEdge) * dvEdge(iEdge) * invAreaCell1
+             factor = 0.5_RKIND * dcEdge(iEdge) * dvEdge(iEdge)
```

Comment:
@vanroekel thanks for the detective work here. Summing the factors and dividing by the sum guarantees that we are not off by a factor, so that is a nice choice. I will add a commit where we remove all these factors of 0.5, which are not needed when we normalize by the sum anyway.

Reply:
@mark-petersen, please, please don't do that! The efficiency gain (if any) is not worth the confusion this causes about what the real area is. This confusion can then propagate to other code, as I believe happened here.

I would like us to do precisely the opposite and make sure areas are correct even when normalization makes this unnecessary. This confusion is just pointless. I can't imagine it saves any meaningful flops.